database/database.go

```diff
@@ -0,0 +1,154 @@
+package database
+
+import (
+	"database/sql"
+	"strings"
+
+	"github.com/aybabtme/log"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Config struct {
+	Path string
+	Log  *log.Log
+}
+
+type DB struct {
+	Config *Config
+	SQL    *sql.DB
+}
+
+type Points struct {
+	From, To, Reason string
+	Points           int
+}
+
+// The Leaderboard lists the top X users
+type Leaderboard []*User
+
+// User is an entry in the Leaderboard
+type User struct {
+	Name   string
+	Points int
+}
+
+func New(config *Config) (*DB, error) {
+	instance := &DB{
+		Config: config,
+	}
+
+	err := instance.Init()
+
+	if err != nil {
+		return nil, err
+	}
+
+	return instance, nil
+}
+
+func (db *DB) Init() error {
+	sqlite, err := sql.Open("sqlite3", db.Config.Path)
+
+	if err != nil {
+		return err
+	}
+
+	db.SQL = sqlite
+	return db.createTable()
+}
+
+func (db *DB) createTable() error {
+	schema := `
+		create table if not exists karma (
+		^id^ integer primary key,
+		^from^ text not null,
+		^to^ text not null,
+		^points^ integer not null,
+		^reason^ text,
+		^timestamp^ text not null default (datetime('now'))
+	)`
+	schema = strings.Replace(schema, "^", "`", -1)
```

Comment:
Maybe just put the string literal inside the `strings.Replace` call?

Your choice, but at least fix the indentation of `create table` with respect to the things inside it. I note that you have a de-dented `)` on the last line.